### ISSUE TITLE
Using Dorado with SQK-RBK114-24 with FLG114 flowcell?

### ISSUE THREAD

**Issue by tchrisboles** [nanoporetech/dorado#664]
# Does Dorado work with SQK-RBK114-24 barcoded libraries on Flongle flow cells?
## I installed new Dorado, but can't get it to basecall a barcoded run using a Flongle FLG114 flow cell. What am I missing?
*Please provide a clear and concise description of the issue you are seeing and the result you expect.*

## Steps to reproduce the issue:

*Please list any steps to reproduce the issue.*

## Run environment:
- Dorado version: 0.5.3-linux-x64
- Dorado command:

![Screenshot 2024-03-01 145452](https://github.com/nanoporetech/dorado/assets/33500125/5f0881a2-71d4-4615-93e5-20c20d54c6e3)

- Operating system: Ubuntu 22.04
- Hardware (CPUs, Memory, GPUs): HP-Z8-G4 workstation with RTX2080Ti GPU
- Source data type - pod5 
- Source data location - hard drive
- Details about data (flow cell, kit, read lengths, number of reads, total dataset size in MB/GB/TB): 
- Dataset to reproduce, if applicable (small subset of data to share as a pod5 to reproduce the issue):

## Logs

* Please provide output trace of dorado (run dorado with -v, or -vv on a small subset)

**Comment by tchrisboles** [comment]
Figured it out. 
I mistakenly thought the Dorado install package would bring in the current basecalling models. 
After I downloaded the models, everything worked fine.

**Comment by HalfPhoton** [comment]
Hi @tchrisboles  - thanks for the update.

We should able to detect the `FLG114` and `SQK-RBK114-24` as the r10.4 chemistry automatically. 
Can you please report the verbose logging output when trying to use the auto model selection by adding `-vv`?

This would be helpful to see what went wrong in the first instance.

Kind regards,
Rich

**Comment by tchrisboles** [comment]
Sorry, a bit of a novice on linux here -- I don't know how to redirect the screen output for the -vv option while writing the bam outputs.

I reran the dorado basecaller using the "sup" option instead of the full model name "dna_r10.4.1_e8.2_400bps_sup@4.3.0", and the command ran fine. The file sizes of the bc'd libraries match what I got using the full model name in the command. 

I assume this means that after installation of the dorado package, one needs to manually download the models before running the basecaller. Maybe the github instructions already say this and I missed it.

Hope this helps.

![image](https://github.com/nanoporetech/dorado/assets/33500125/785e3f17-fed4-4ceb-9599-6f7420d0fb21)

**Comment by HalfPhoton** [comment]
@tchrisboles, dorado should have downloaded the `sup` model automatically for you but was unable to determine the sequencing chemistry and therefore the type of model to download from the data that was given. 
I would expect that you'd see the same error even after downloading the model. Has the data changed since the first error and now?

You shouldn't need to download the model anymore but something strange was happening in the first instance which is what I'd like to figure out here. 

Kind regards,
Rich

**Comment by tchrisboles** [comment]
Rich, sorry not to be very much help on the model selection errors I described in my initial post.

After looking at my bash history. I see that after I performed "dorado download --model all" I still had a couple of failed basecaller commands see commands 1904 and 1905:

![image](https://github.com/nanoporetech/dorado/assets/33500125/8f0de119-09d3-4d99-b895-b45ebc694576)

I must have then realized I hadn't used the "-r" flag to descend into pod5_pass subdirectories. That command must have succeeded because in the next command ("1906"), I used the -r flag, and then I went straight on to the demux step in "1907". 

I still have no explanation for the errors in my initial post which seem to be related to basecaller model selection, and I don't know how to reconstruct that error.

So the initial failures were probably not due to
